node/core/pvf/src/host.rs

```diff
@@ -86,6 +115,39 @@ impl ValidationHost {
 			.await
 			.map_err(|_| "the inner loop hung up".to_string())
 	}
+
+	/// Utility function enabling to convert the ValidationResult buffer back to Validation Result
+	/// 
+	/// This function enables the implementation of the Candidate Validation subsystem to delineate
+	/// the entry-point of the PVF and the oneshot channel passed to the execution of any function
+	/// exported in the pvf.
+	/// 
+	/// Returns an error if the worker has generated an invalid Validation Result buffer
+	pub fn parse_validation(&self, input: Vec<u8>) -> Result<ValidationResult, ValidationError> {
+		use parity_scale_codec::Decode;
+		ValidationResult::decode(&mut &input[..]).map_err(|_| {
+			ValidationError::InvalidCandidate(
+				InvalidCandidate::WorkerReportedError("Worker produced invalid ValidationResult".to_string())
+			)
+		})
+	}
+
+
+	/// Utility function enabling to convert the ValidationResult buffer back to Validation Result
+	/// 
+	/// This function enables the implementation of the Candidate Validation subsystem to delineate
+	/// the entry-point of the PVF and the oneshot channel passed to the execution of any function
+	/// exported in the pvf.
+	/// 
```

Comment:
There is quite some trailing whitespaces. 

Could you please clean them up in this PR. Also it would be great if you can configure your editor to remove trailing whitespaces automatically on save